Odata-docs/webapi-8/fundamentals/server-driven-paging.md

```diff
@@ -0,0 +1,268 @@
+---
+title:  "Server-driven Paging in ASP.NET Core OData 8"
+description: Server-driven paging in ASP.NET Core OData 8.
+date:   2023-01-11
+ms.date: 1/11/2023
+author: habbes
+ms.author: clhabins
+---
+
+# Server-driven Paging in ASP.NET Core OData 8
+
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+In [server-driven paging](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_ServerDrivenPaging), the server returns the first page of results. If total number of results is greater than the page size, the server returns the first page along with a "next link" that can be used to fetch the next page of results. Each subsequent page should include a next link except for the last page. Using this approach, the client needs to fetch pages sequentially. It cannot jump to an arbitrary page. The next link is included in the response using the [`@odata.nextLink`](http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_ControlInformationnextLinkodatanextL) annotation.
```

Comment:
you've provided a link to the v4.01 format but you are using a v4.0 format.

Reply:
Thanks. Updated.